frontend/src/views/Bridge.vue

```diff
@@ -29,6 +29,12 @@
               {{$t('bridge.showCharacters')}}
             </b-button>
           </div>
+          <div class="p-2">
+            <b-button variant="primary" @click="nftType = 'shield'; selectedNftId = ''" class="gtag-link-others"
+              tagname="show_shields_bridge" :disabled="nftType === 'shield'">
+              Show Shields
```

Comment:
Missing translation

Reply:
resolved in 921a343f093d493a8de431cf85e0ba426d0eeb73